src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, serif;
+}
+
+body {
+  margin: 0;
+}
+
+header {
```

Comment:
check out point 1 in the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)